desktop/src/main/java/bisq/desktop/main/portfolio/duplicateoffer/DuplicateOfferDataModel.java

```diff
@@ -85,6 +87,12 @@ public void populateData(Offer offer) {
         setPrice(offer.getPrice());
         setVolume(offer.getVolume());
         setUseMarketBasedPrice(offer.isUseMarketBasedPrice());
+
+        if (offer.getBuyerSecurityDeposit().value == Restrictions.getMinBuyerSecurityDepositAsCoin().getValue())
+            setBuyerSecurityDeposit(Restrictions.getMinBuyerSecurityDepositAsPercent());
+        else
+            setBuyerSecurityDeposit(CoinUtil.getAsPercentPerBtc(offer.getBuyerSecurityDeposit(), offer.getAmount()));
+
```

Comment:
NIT: we decided to always use brackets, even for one-liners as we had issues in the past.